feg/cloud/go/services/feg/obsidian/models/defaults.go

```diff
@@ -183,6 +185,13 @@ func newDefaultS6a() *S6a {
 	}
 }
 
+func newDefaultS8() *S8 {
+	return &S8{
+		LocalAddress: "10.0.0.1",
+		PgwAddress:   "10.0.0.2",
```

Comment:
I am not sure if it's a default which should work universally. Maybe "localhost" for LocalAddress?